src/Font/UiFontDescriptor.cc

```diff
@@ -0,0 +1,57 @@
+#include "../../ui.h"
+#include "../ui-node.h"
+#include "nbind/nbind.h"
+
+UiFontDescriptor::UiFontDescriptor(uiFontDescriptor * desc) {
+	d = desc;
+	cleanup = 1;
+}
+
+UiFontDescriptor::UiFontDescriptor(char *family, double size, int weight, int italic, int stretch) {
+    d = new uiFontDescriptor();
+    d->Family = family;
+    d->Size = size;
+    d->Weight = weight;
+    d->Italic = italic;
+    d->Stretch = stretch;
+}
+
+void UiFontDescriptor::free() {
+	if(cleanup){
```

Comment:
If cleanup is a flag to guard by multiple calls of the free method, I guess you want to set it to 0

Reply:
> If cleanup is a flag to guard by multiple calls of the free method, I guess you want to set it to 0

Ok, gorget this....